dts/arm/st/h5/stm32h5.dtsi

```diff
@@ -491,6 +491,54 @@
 			status = "disabled";
 		};
 
+		i2s1: i2s@40013000 {
+			compatible = "st,stm32h7-i2s", "st,stm32-i2s";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x40013000 0x400>;
+			clocks = <&rcc STM32_CLOCK_BUS_APB2 0x00001000>,
+				<&rcc STM32_SRC_PLL1_Q SPI1_SEL(0)>;
+			dmas = <&gpdma1 0 7 STM32_DMA_PERIPH_TX
+				&gpdma1 1 6 (STM32_DMA_PERIPH_RX | STM32_DMA_MEM_INC | \
+					STM32_DMA_PERIPH_16BITS | STM32_DMA_MEM_16BITS | \
```

Comment:
Should we use `STM32_DMA_PERIPH_16BITS | STM32_DMA_MEM_16BITS` in TX as well ?
If not, let's add a comment to explain this configuration.

Nit: A new flag `STM32_DMA_16BITS` could be added as a short to `STM32_DMA_PERIPH_16BITS | STM32_DMA_MEM_16BITS`

Reply:
Yes , it should be used also in TX